### PR TITLE
Switch from chimad-phase-field to pfhub repo

### PR DESCRIPTION
**Pull request by wd15**
Address #556

Switched from chimad-phase-field to pfhub repository.

 - Removed a number of hard links and placed in the config

 - Should be no more references to chimad-phase-field

 - References to pfhub are only in locations that Jekyll doesn't write
   to

Need to update Travis CI and check that Staticman still works after merge.

These changes can be [viewed live][live-site].

**Remember to tear down the [live site][live-site] when merging or
  closing this pull-request.**

[live-site]: http://random-cat-681.surge.sh

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/pfhub/681)

### CODE REVIEW

**Comment by wd15** [comment]
@tkphd, can you review this quickly as it's required to get the site up again.

**Comment by tkphd** [comment]
@wd15 working on it now

**Comment by tkphd** [comment]
Reviewed 27 of 27 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/usnistgov/pfhub/681)*

File: mailing_list.md [hunk 1]
```diff
@@ -9,25 +9,25 @@ to sign up for the mailing list by sending a subscription email:
...
 
-    chimad-phase-field+unsubscribe@list.nist.gov
+    {{ site.links.maillist_unsubsribe }}
 
 ### List Archive
 
 The list is archived as a Google Group, see
 
-<https://groups.google.com/a/list.nist.gov/forum/#!forum/chimad-phase-field>
+{{ site.links.maillist_archive }}
```
Comment (tkphd) [inline mailing_list.md]: URL renders as plaintext. Requires hyperlink wrapper `[]()`.

File: mailing_list.md [hunk 2]
```diff
@@ -9,25 +9,25 @@ to sign up for the mailing list by sending a subscription email:
 
 To:
```
Comment (tkphd) [inline mailing_list.md]: Recommend formatting this as a quote block, but that's a stylistic decision.

**Comment by tkphd** [comment]
The "Open Chat" button does not work on surge (box opens, prints "This is not the chat you're looking for"). It *does* work on pages.nist.gov/pfhub, so it's probably not an issue?

**Pull request merged**